earthpy/tests/test_io.py

```diff
@@ -220,6 +220,15 @@ def test_url_download_txt_file_with_content_disposition(eld):
     assert path.endswith("abc.txt") and os.path.isfile(path)
 
 
+@skip_on_ci
+@pytest.mark.vcr()
```

Comment:
This test captures the log itself. but what if we want to capture that the log message prints to the console if verbose=True?